### ISSUE TITLE
Not able to update even via cli, got error tls: failed to verify certificate: x509: certificate signed by unknown authority

### ISSUE THREAD

**Issue by KetanM94** [AdguardTeam/AdGuardHome#7002]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Custom (please mention in the description)

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

0.107.48

### Action

I'm running AdGuardHome in Termux on an Android device(ARMv7). I'm using Linux ARMv7 build.
I am unable to update via Web UI. So I tried using the cli command to update. But got `tls: failed to verify certificate: x509: certificate signed by unknown authority` error. I can't update the filters as well.

### Expected result

```
~ $ AdGuardHome/AdGuardHome --update
2024/05/16 03:44:32.463976 [info] AdGuard Home, version v0.107.48
2024/05/16 03:44:32.485535 [info] tls: using default ciphers
2024/05/16 03:44:32.491837 [info] safesearch default: disabled
2024/05/16 03:44:32.496500 [info] dnsproxy: cache: enabled, size 4096 b
2024/05/16 03:44:32.496893 [info] dnsproxy: max goroutines is set to 300
2024/05/16 03:44:32.498506 [info] dnsproxy: cache: disabled
2024/05/16 03:44:32.498720 [info] Ratelimit is enabled and set to 100 rps, IPv4 subnet mask len 24, IPv6 subnet mask len 56
2024/05/16 03:44:32.498797 [info] dnsproxy: server will refuse requests of type ANY
2024/05/16 03:44:32.498852 [info] dnsproxy: cache: enabled, size 4000000 b
2024/05/16 03:44:32.498908 [info] dnsproxy: max goroutines is set to 300
2024/05/16 03:44:32.499129 [info] cmdline update: performing update
2024/05/16 03:44:32.499323 [info] clients: processing addresses
2024/05/16 03:44:33.129739 [info] no updates available
```

### Actual result

```
~ $ AdGuardHome/AdGuardHome --update
2024/05/16 03:37:43.516501 [info] AdGuard Home, version v0.107.48
2024/05/16 03:37:43.555290 [info] tls: using default ciphers
2024/05/16 03:37:43.562743 [info] safesearch default: disabled
2024/05/16 03:37:43.569751 [info] dnsproxy: cache: enabled, size 4096 b
2024/05/16 03:37:43.570219 [info] dnsproxy: max goroutines is set to 300
2024/05/16 03:37:43.575992 [info] dnsproxy: cache: disabled
2024/05/16 03:37:43.576614 [info] Ratelimit is enabled and set to 100 rps, IPv4 subnet mask len 24, IPv6 subnet mask len 56
2024/05/16 03:37:43.576926 [info] dnsproxy: server will refuse requests of type ANY
2024/05/16 03:37:43.577298 [info] dnsproxy: cache: enabled, size 4000000 b
2024/05/16 03:37:43.577598 [info] dnsproxy: max goroutines is set to 300
2024/05/16 03:37:43.578408 [info] cmdline update: performing update
2024/05/16 03:37:43.578600 [info] clients: processing addresses
2024/05/16 03:37:43.942643 [error] getting version info from https://static.adtidy.org/adguardhome/release/version.json: updater: HTTP GET https://static.adtidy.org/adguardhome/release/version.json: Get "https://static.adtidy.org/adguardhome/release/version.json": tls: failed to verify certificate: x509: certificate signed by unknown authority
```

**Comment by KetanM94** [comment]
I tested with version 0.107.46 and can do update through both Web UI and cli. Seems like a regression.
I was able to update with

**Comment by FNsi** [comment]
How? You think u can do it but in fact it's not working . 

I updated the last release (108 version) today to use quic in my old 3.xx kernel, but it didn't work, 


and I feel so bad it made x509 unknown authority even in my normal dns server queries  

What a surprise 🙅

**Comment by KetanM94** [comment]
Are you running in Termux?
Then you can just run like
`SSL_CERT_DIR="/system/etc/security/cacerts/" /path/to/AdGuardHome`

If running as root in Termux then
`sudo SSL_CERT_DIR="/system/etc/security/cacerts/" su -c /path/to/AdGuardHome`

**Comment by FNsi** [comment]
> If running as root in Termux then
> `sudo SSL_CERT_DIR="/system/etc/security/cacerts/" su -c /path/to/AdGuardHome`

Not working for me. The problem is I was facing that one month ago, before updating 
. But I can query the dns.

And after, no .

**Comment by KetanM94** [comment]
> 
> 
> > If running as root in Termux then
> > `sudo SSL_CERT_DIR="/system/etc/security/cacerts/" su -c /path/to/AdGuardHome`
> 
> Not working for me. The problem is I was facing that one month ago, before updating 
> . But I can query the dns.
> 
> And after, no .
> 
> 
> 

Then your issue might be different. You can try version 0.107.46. I don't know if that supports quic.

**Comment by FNsi** [comment]
I removed the adgh from my our phone.


107.43 or older may somehow work(the one I used before today, I tried a lot updates in January, like situation I face today. Turn out I should use the very early version till the world end....)
Thanks for your help!

**Comment by ainar-g** [comment]
@KetanM94, we do not support running AGH on Android, and the error seem to indicate an issue with the local TLS certificate storage.  Please make sure that your root certificates are updated.

Also, in [Go 1.22](https://tip.golang.org/doc/go1.22) that directory seems to be included by default:

 >  On Android, root certificates will now be loaded from `/data/misc/keychain/certs-added` as well as `/system/etc/security/cacerts`.

**Comment by KetanM94** [comment]
> @KetanM94, we do not support running AGH on Android, and the error seem to indicate an issue with the local TLS certificate storage.  Please make sure that your root certificates are updated.
> 
> Also, in [Go 1.22](https://tip.golang.org/doc/go1.22) that directory seems to be included by default:
> 
>  >  On Android, root certificates will now be loaded from `/data/misc/keychain/certs-added` as well as `/system/etc/security/cacerts`.

Yes. I suspected the same and updated the root certificates. But even after that I got same error.
I've been running AGH on Android since last 3 years with no issues. Last working version was 0.107.46
Will there be no support for Android in future?

**Comment by Stockgolm** [comment]
Can you write a guide how to run adguard home on termux[root]
